rewrite-toml/src/main/java/org/openrewrite/toml/tree/Toml.java

```diff
@@ -309,9 +314,17 @@ class Table implements TomlValue {
         @With
         Markers markers;
 
-        @With
+        @Nullable
```

Comment:
The table name is a required element otherwise it would result in an invalid Toml document.

Reply:
In the parser I think we pass in `null` for inline tables.

Reply:
https://github.com/openrewrite/rewrite/blob/02411bd6e386f793b8c8af2c49f2154b528f6144/rewrite-toml/src/main/java/org/openrewrite/toml/internal/TomlParserVisitor.java#L370-L376

Reply:
Ahh, right, right. I forgot about that piece.